src/owl-core-lib/connectors-owl-core.xsl

```diff
@@ -93,107 +94,90 @@
-                    f:buildURIFromElement(./source/role, false(), fn:true())
-                else
-                    ()"/>
+                for $connector in $connectorsWithSameName
+                return
+                    if ($connector/documentation/@value) then
+                        fn:concat($connector/documentation/@value, ' (', f:getConnectorName($connector), ') ')
+                    else
+                        ()"/>
 
-        <xsl:variable name="connectorDocumentation"
-            select="f:formatDocString(./documentation/@value)"/>
+        <xsl:variable name="documentation"
+            select="f:formatDocString(fn:string-join($connectorDocumentations))"/>
 
-        <xsl:if test="$targetRoleURI">
-            <xsl:variable name="name" select="f:lexicalQNameToWords(./target/role/@name)"/>
-            <xsl:variable name="note" select="f:formatDocString(./target/documentation/@value)"/>
+        <xsl:variable name="connectorNotes" as="xs:string*"
+            select="
+                for $connector in $connectorsWithSameName
+                return
+                    if ($connector/target/role/@name = $connectorName and $connector/target/documentation/@value) then
+                        fn:concat($connector/target/documentation/@value, ' (', f:getConnectorName($connector), ') ')
+                    else
+                        if ($connector/source/role/@name = $connectorName and $connector/source/documentation/@value) then
+                            fn:concat($connector/source/documentation/@value, ' (', f:getConnectorName($connector), ') ')
+                        else
+                            ()"/>
 
-            <owl:ObjectProperty rdf:about="{$targetRoleURI}">
-                <rdfs:label xml:lang="en">
-                    <xsl:value-of select="$name"/>
-                </rdfs:label>
-                <skos:prefLabel xml:lang="en">
-                    <xsl:value-of select="$name"/>
-                </skos:prefLabel>
-                <xsl:if test="$connectorDocumentation">
-                    <skos:definition xml:lang="en">
-                        <xsl:value-of select="$connectorDocumentation"/>
-                    </skos:definition>
-                    <rdfs:comment xml:lang="en">
-                        <xsl:value-of select="$connectorDocumentation"/>
-                    </rdfs:comment>
-                </xsl:if>
-                <xsl:if test="$note">
-                    <skos:scopeNote xml:lang="en">
-                        <xsl:value-of select="$note"/>
-                    </skos:scopeNote>
-                    <rdfs:comment xml:lang="en">
-                        <xsl:value-of select="$note"/>
-                    </rdfs:comment>
-                </xsl:if>
-                <rdfs:isDefinedBy rdf:resource="{$base-uri}"/>
-            </owl:ObjectProperty>
-        </xsl:if>
+        <xsl:variable name="note" select="f:formatDocString(fn:string-join($connectorNotes))"/>
```

Comment:
by the way!!!

All the definitions/notes from any element (connector, attribute, class, enumeration) must be filtered/formatted using the function `f:formatDocString`. The reason is funky formatting used by the EA and it is not according to HTML standards and could lead to encoding problems.

Reply:
created new ticket